packages/safe-core-sdk/src/utils/transactions/gas.ts

```diff
@@ -1,6 +1,6 @@
 import { BigNumber } from 'ethers'
 import { GnosisSafe } from '../../../typechain'
-import SafeTransaction from './SafeTransaction'
+import SafeTransaction, { OperationType } from './SafeTransaction'
 
 function estimateDataGasCosts(data: any): number {
```

Comment:
This function won't work for anything other than a string (or anything that defines `.match`), why does it use any type for its param?